### ISSUE TITLE
Keyboard does not appear, by any means

### ISSUE THREAD

**Issue by WajidAli007** [ChaosLeong/PinView#13]
Hi,
I have tried many methods just to bring the keyboard, but it doesn't appear. I used its method to showSoftInput but still nothing.
Does any one have got the same problem?

**Comment by ChaosLeong** [comment]
Can you tell me your phone model and Android version?

**Comment by privacy-unknown** [comment]
@WajidAli007 ensure that you have not missed the PinView style

*Step 2 - in setup
https://github.com/ChaosLeong/PinView#step-2

```
<com.chaos.view.PinView
    android:id="@+id/pinView"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    style="@style/PinWidget.PinView" />
```

**Comment by ChaosLeong** [comment]
The issue has been fixed in version 1.3.1.
